### DISCUSSION TITLE
Build failed

### DISCUSSION THREAD

**Discussion by hussamkhaled** [codemagic-ci-cd/codemagic-docs#2134] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

React Native

### Steps to reproduce

I am trying to build my react native app on a mac mini m1 to push my second version. While running,  
- name: Build ipa for distribution
           script: |
              xcode-project build-ipa \
                --workspace "$CM_BUILD_DIR/ios/$XCODE_WORKSPACE" \
                --scheme "$XCODE_SCHEME"

i got an error and can't know what to do 

### Expected results

I expected my app to be build as soon as possible because i had deadlines 

### Actual results

I got this error

Failed to archive /Users/builder/clone/ios/NuFmMobile.xcworkspace. The following build commands failed:

CompileC /Users/builder/Library/Developer/Xcode/DerivedData/NuFmMobile-gnduspqjcmofzmbtwqjidebebcmx/Build/Intermediates.noindex/ArchiveIntermediates/NuFmMobile/IntermediateBuildFilesPath/Pods.build/Release-iphoneos/RCT-Folly.build/Objects-normal/arm64/json.o /Users/builder/clone/ios/Pods/RCT-Folly/folly/json.cpp normal arm64 c++ com.apple.compilers.llvm.clang.1_0.compiler (in target 'RCT-Folly' from project 'Pods')
	...
	/Users/builder/clone/ios/Pods/Pods.xcodeproj: warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 11.0, but the range of supported deployment target versions is 12.0 to 17.0.99. (in target 'RNCAsyncStorage' from project 'Pods')
	/Users/builder/clone/ios/Pods/Pods.xcodeproj: warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 11.0, but the range of supported deployment target versions is 12.0 to 17.0.99. (in target 'RCT-Folly' from project 'Pods')
	warning: Run script build phase '[CP-User] Generate app.manifest for expo-updates' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'EXUpdates' from project 'Pods')
	warning: Run script build phase '[CP-User] Generate app.config for prebuilt Constants.manifest' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'EXConstants' from project 'Pods')
	/Users/builder/clone/ios/Pods/Pods.xcodeproj: warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 11.0, but the range of supported deployment target versions is 12.0 to 17.0.99. (in target 'BEMCheckBox' from project 'Pods')
	/Users/builder/clone/ios/Pods/Pods.xcodeproj: warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 11.0, but the range of supported deployment target versions is 12.0 to 17.0.99. (in target 'ASN1Decoder' from project 'Pods')

**Comment by himesh-cm** [comment]
Hi @hussamkhaled,
Please share a build id so we can further investigate the issue.

**Comment by himesh-cm** [comment]
Hi @hussamkhaled , I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
